applications/nrf_desktop/CMakeLists.txt

```diff
@@ -47,6 +47,61 @@ if(CONFIG_IMG_MANAGER)
   zephyr_library_link_libraries(MCUBOOT_BOOTUTIL)
 endif()
 
+if(CONFIG_BOOTLOADER_MCUBOOT)
+  if(NOT (CONFIG_MCUBOOT_BOOTLOADER_MODE_SINGLE_APP OR
+          CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP OR
```

Comment:
`mcuboot_qspi` configuration of `nrf52840dk` uses MCUboot with swap (`CONFIG_MCUBOOT_BOOTLOADER_MODE_SWAP_USING_MOVE`). It triggers the warning here (false positive).